### PR TITLE
Feat: NER Regex extractor: Use first capturing group if exists

### PR DESCRIPTION
**Pull request by danlobo**
## PR Checklist

- [ X ] I have run `npm test` locally and all tests are passing.
- [ X ] I have added/updated tests for any new behavior.
- [ X ] If this is a significant change, an issue has already been created where the problem / solution was discussed: [N/A, or add link to issue here]

## PR Description

Small change to allow NER regex extractor to use the first capturing group, if exists.

### CODE REVIEW

File: packages/ner/src/extractor-regex.js [hunk 1]
```diff
@@ -44,12 +44,22 @@ class ExtractorRegex {
     do {
       const match = regex instanceof RegExp ? regex.exec(utterance) : null;
       if (match) {
-        result.push({
-          start: match.index,
-          end: regex.lastIndex - 1,
-          accuracy: 1,
-          sourceText: match[0],
-        });
+        if (match.length == 1) {
```
Comment (ericzon) [inline packages/ner/src/extractor-regex.js:47]: Please @danlobo  use === instead of ==

Comment (danlobo) [reply packages/ner/src/extractor-regex.js]: Done. Thanks for the review.

**Comment by ericzon** [comment]
Please @danlobo fix these conflicts and we'll be ready to go

**Comment by ericzon** [comment]
@danlobo  even better, these package-locks changes are outside of the scope of this PR, better remove them to avoid any unnecessary conflict.
packages/builtin-compromise/package-lock.json
packages/fb-connector/package-lock.json
packages/lang-ja/package-lock.json
packages/utils/package-lock.json

**Comment by danlobo** [comment]
ok, reverted the package-locks, never needed to change it in first place. Thanks for the heads up.

**Pull request merged**